pages/favorite.js

```diff
@@ -0,0 +1,5 @@
+import FavoriteListPage from "@/components/FavoriteList";
+
+export default function FavoritesPage({ liked, toggleLiked }) {
+  return <FavoriteListPage liked={liked} toggleLiked={toggleLiked} />;
```

Comment:
use ActivityList here, fetch the activities with useSWR (as you did in the List component) and don't forget to filter the activities here before passing them as "values" or "activities" to the List component.